packages/core/src/forms/utils/validate.ts

```diff
@@ -8,19 +8,18 @@ import { CdsControlMessage } from '../control-message/control-message.element';
 import { CdsControl } from '../control/control.element';
 
 // https://developer.mozilla.org/en-US/docs/Web/API/ValidityState
-export enum ValidityStateKey {
-  valueMissing = 'valueMissing',
-  badInput = 'badInput',
-  customError = 'customError',
-  patternMismatch = 'patternMismatch',
-  rangeOverflow = 'rangeOverflow',
-  rangeUnderflow = 'rangeUnderflow',
-  stepMismatch = 'stepMismatch',
-  tooLong = 'tooLong',
-  tooShort = 'tooShort',
-  typeMismatch = 'typeMismatch',
-  valid = 'valid',
-}
+export type ValidityStateKey =
+  | 'valueMissing'
+  | 'badInput'
+  | 'customError'
+  | 'patternMismatch'
+  | 'rangeOverflow'
+  | 'rangeUnderflow'
+  | 'stepMismatch'
+  | 'tooLong'
+  | 'tooShort'
+  | 'typeMismatch'
+  | 'valid';
```

Comment:
Changed to be a type instead of a enum. Generates less runtime code. Easier to use as a consumer by being allowed to pass the string instead of requiring to use the enum. Caught this when using the React wrappers.